app/views/events/show.html.erb

```diff
@@ -60,9 +60,13 @@
         data: { confirm: 'Are you sure? This will also attempt to delete the Discord message related to this event.' },
         class: "btn btn-danger" %>
     <% end %>
-    <%= link_to @event.google_calendar_url, class: "btn btn-primary", target: "_blank" do %>
-      <i class="fa fa-calendar"></i> Add to Google Calendar
+   
+    <% if @event.start_date.future? %>
```

Comment:
```suggestion
    <% if @event.end_date.past? %>
```
This way events that are currently happening will still have the button, otherwise LGTM 🚢

Reply:
Oh yeah, I completely forgot that we do have events which last for more than a day. Sounds good sir